### DISCUSSION TITLE
Creating sub lists and breaking out of a sublist

### DISCUSSION THREAD

**Discussion by Gubbi** [Milkdown/milkdown#178] [Q&A]
Hi, 

I'm sure I'm missing something basic. But I'm unable to use tab in a list to indent a level down and use shift + tab to indent a level up within lists. I'm on Mac. I tried here: https://milkdown.dev/#/online-demo

I don't know how to create the `Made By` kind of nested list. Please help.

**Comment by Saul-Mirone** [comment]
By default it's `Cmd + [` and `Cmd + ]`, check keymap here: https://milkdown.dev/#/keyboard-shortcuts.

**Reply by duhaime** [reply]
For others who end up here, I also wanted to indent list items when hitting the tab key, and did so with the following:

```
import { commonmark, listItem, SupportedKeys } from '@milkdown/preset-gfm';

const pluginNodes = commonmark.configure(listItem, {
  keymap: {
    [SupportedKeys.SinkListItem]: 'Tab',
  },
});
```

Then you can use that plugin like any other:

```
const editor = Editor.make()
        .use(nord)
        .use(pluginNodes)
```
